### DISCUSSION TITLE
Latex placement options for link_attributes

### DISCUSSION THREAD

**Discussion by avidseeker** [jgm/pandoc#10369] [Q&A]
In Extension: link_attributes:

```markdown
An inline ![image](foo.jpg){#id .class width=30 height=20px}
```

Output is something like
```
\includegraphics[width=0.5\textwidth,height=\textwidth]{file.jpg}
```

How can I wrap it figure specified by placement options `here, top, bottom, page, etc.`

```tex
\begin{figure}[htbp]
    \includegraphics[width=0.5\textwidth,height=\textwidth]{file.jpg}
\end{figure}
````

**Comment by jgm** [comment]
You'll get a figure if you have just the image in a paragraph.  But I don't think there's any built-in way to add `htbp`.
We could contemplate making this sensitive to a `latex-placement` attribute or something like that?

**Reply by avidseeker** [reply]
My two cents preference is to keep it short using `pos=h` or `position=h`. `latex-placement` is a bit mouthful.

**Reply by SeanESCA** [reply]
Hi, has this been added?

**Reply by jgm** [reply]
No.

**Reply by SeanESCA** [reply]
I need to set the placement for some figures, so something like this would be very helpful. I can work on this if there are no issues with adding this feature, and no work has been started. I think the new attribute is a good way to go about this.
